packages/service/src/event-stream/model/events.ts

```diff
@@ -35,6 +35,8 @@ interface ResumeEvent {
 
 interface SkipEvent {
   id: 'SkipEvent',
+  all: boolean,
```

Comment:
to `all` mi się nie podoba jako nazwa, ale nie wiem jak inaczej

Reply:
🤷

Reply:
może `clearQueue`?